.github/workflows/crossterm_test.yml

```diff
@@ -17,9 +20,20 @@ jobs:
       with:
         rust-version: ${{ matrix.rust }}
     - uses: actions/checkout@master
+    - name: Add Component
```

Comment:
See [this](https://github.com/hecrj/setup-rust-action/blob/master/action.yml#L8-L10). You can use `with.components: 'rustfmt'`.